R-package/README.md

```diff
@@ -97,13 +97,18 @@ After installing `Rtools` and `CMake`, be sure the following paths are added to
     - If you have `Rtools` 4.0, example:
         - `C:\rtools40\mingw64\bin`
         - `C:\rtools40\usr\bin`
+    - If you have `Rtools` 4.2, example:
+        - `C:\rtools42\x86_64-w64-mingw32.static.posix\bin`
+        - `C:\rtools42\user\bin`
```

Comment:
```suggestion
        - `C:\rtools42\usr\bin`
```

Reply:
ha thank you for catching this! Now you know how I pronounce this path in my head 😂